l10n/en-US/viewer.properties

```diff
@@ -268,3 +268,11 @@ editor_ink_canvas_aria_label=User-created image
 # LOCALIZATION NOTE (alt_text_button_label): Alternative text (alt text) helps
 # when people can't see the image.
 alt_text_button_label=Alt text
+editor_alt_text_dialog_label=Choose an option
+editor_alt_text_dialog_description=Alternative text (alt text) helps when people can’t see the image or when it doesn’t load.
```

Comment:
I see that this is the string that was suggested in the specification, however the "or when it doesn’t load" part seems slightly strange in the context of a PDF document (where the image will be embedded) and it almost feels like this is directly taken from a web use-case where images may indeed not load?

Reply:
Marco raised the exact same question few days ago and the answer was that we keep this string for the moment (it could change in the future).

Reply:
So multiple people have, independently of each other, pointed out that (part of) the string doesn't really make sense in a PDF-context and UX still want's to keep it?

> (it could change in the future).

I very much hope so, but given the saying "nothing is more permanent than a temporary solution" I'm not holding my breath here :-)

Reply:
The answer from UX was that the goal is to educate people about using "alt text" in general, and we want to reduce confusion for them (so they don't have to think that the "alt text" meaning is different in different contexts).